gap/matrix.gi

```diff
@@ -667,6 +667,8 @@ InstallGlobalFunction( FindKernelLowerLeftPGroup,
   function(ri)
     local b,curlay,done,el,f,i,l,lens,lvec,nothingnew,pivots,pos,ready,
           rifac,s,v,x,y;
+
+    Info(InfoRecog, 2, "Running FindKernelLowerLeftPGroup...");
```

Comment:
Is this here on purpose? Just wondering since it's not mentioned in the commit msg.

Reply:
It's in there by accident, but I did mean to add it to recog eventually. It may not be "pure", but I'd just merge it along now, no point in wasting time just to get that one line into a separate commit / separate PR... :-)